LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/BasicsStation/Processors/LnsProtocolMessageProcessor.cs

```diff
@@ -232,6 +232,13 @@ internal async Task InternalHandleDataAsync(RouteValueDictionary routeValues, We
                     break;
                 case var messageType and (LnsMessageType.DownlinkMessage or LnsMessageType.RouterConfig):
                     throw new NotSupportedException($"'{messageType}' is not a valid message type for this endpoint and is only valid for 'downstream' messages.");
+                case var messageType and (LnsMessageType.ProprietaryDataFrame
+                                          or LnsMessageType.MulticastSchedule
+                                          or LnsMessageType.TimeSync
+                                          or LnsMessageType.RunCommand
+                                          or LnsMessageType.RemoteShell):
+                    this.logger.LogWarning($"'{messageType}' ({messageType.ToBasicStationString()}) is not handled in current LoRaWan Network Server implementation.");
```

Comment:
How often do we expect this to happen? Is it really worth logging a warning here, or is it expected and maybe a `Debug` statement would suffice to avoid flooding the logs?

Reply:
Frequency should not be high. I think it's important to keep it as a warning to make sure that anyone expecting these messages to be handled, actually realizes that's not the case